examples/src/bin/tlsserver-mio.rs

```diff
@@ -551,15 +555,29 @@ fn load_ocsp(filename: &Option<String>) -> Vec<u8> {
     ret
 }
 
+fn load_crls(filenames: &Vec<String>) -> Vec<UnparsedCertRevocationList> {
+    let mut ret = Vec::new();
+
+    for filename in filenames {
+        let mut der = Vec::new();
+        fs::File::open(filename)
+            .expect("cannot open CRL file")
+            .read_to_end(&mut der)
+            .unwrap();
+        ret.push(UnparsedCertRevocationList(der));
+    }
+
+    ret
+}
+
 fn make_config(args: &Args) -> Arc<rustls::ServerConfig> {
     let client_auth = if args.flag_auth.is_some() {
         let roots = load_certs(args.flag_auth.as_ref().unwrap());
         let mut client_auth_roots = RootCertStore::empty();
         for root in roots {
             client_auth_roots.add(&root).unwrap();
         }
-        // TODO(@cpu): Parse CRLs from Args.
-        let crls = Vec::default();
+        let crls = load_crls(&args.flag_crl);
```

Comment:
Can we avoid this indirection in the commit history? It doesn't seem useful to change the examples twice like this.

Reply:
Sure, I'll fold the commit that adds the command line flag processing into this commit.